### PR TITLE
Remove 1559 flag v2

### PR DESCRIPTION
**Pull request by adonesky1**
Parallel version of https://github.com/MetaMask/metamask-extension/pull/11596, with fixed tests. Here in parallel for comparison to ensure that all desired changes are present.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 1]
```diff
@@ -265,39 +250,25 @@ export default class ConfirmTransactionBase extends Component {
...
-  // TODO: rename this 'handleCloseEditGas' later when we remove the
-  // SHOW_EIP_1559_UI flag/branch
-  handleCloseNewGasPopover() {
+  handleCloseEditGas() {
     this.setState({ editingGas: false });
   }
 
   renderDetails() {
     const {
-      primaryTotalTextOverride,
```
Comment (danjm) [inline ui/pages/confirm-transaction-base/confirm-transaction-base.component.js:283]: primaryTotalTextOverride and secondaryTotalTextOverride need to remain

Comment (darkwing) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js:283]: These were added to the end of the object -- they're still here

Comment (danjm) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js:283]: yes, my mistake!

**Review by danjm — DISMISSED** [review]
Nice work!

**Review by danjm — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**